tests/integration/block_create_test.go

```diff
@@ -107,21 +106,18 @@ func (s *BlockCreateSuite) TestCreateSamePVCTwice() {
 	s.testClient.BlockClient.List(s.namespace)
 
 	logger.Infof("create pool and storageclass")
-	result0, err0 := s.testClient.PoolClient.Create(poolName, s.namespace, 1)
-	checkOrderedSubstrings(s.T(), result0, poolName, "created")
+	err0 := s.testClient.PoolClient.Create(poolName, s.namespace, 1)
```

Comment:
What about renaming the error vars to just `err` in this PR too?

Reply:
Ok, i updated the errors in this file, but would like to hold off looking through other test sources as well. Definitely something worth cleaning up though...